src/api/profile/content-types/profile/schema.json

```diff
@@ -44,6 +44,18 @@
       "config": {
         "provider": "azure-storage"
       }
+    },
+    "location": {
+      "type": "string"
+    },
+    "role": {
+      "type": "string"
+    },
+    "headline": {
+      "type": "string"
+    },
+    "website": {
```

Comment:
What kind of website will this be? Someone's portfolio, LinkedIn, etc? Maybe this can be a list?

Reply:
This will be a portfolio website. We can add a separate linkedin later.

Reply:
Maybe we can call this field `portfolioWebsite`.